packages/-ember-data/tests/integration/application-test.js

```diff
@@ -1,7 +1,7 @@
 import Application from '@ember/application';
 import Namespace from '@ember/application/namespace';
 import Controller from '@ember/controller';
-import Service, { inject as service } from '@ember/service';
```

Comment:
I suspect this needs to drop the `Service, ` portion and do `const Service = s.default` below